website_medical/static/src/js/file_upload_text_insert.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016 LasLabs Inc.
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+$(document).ready(function() {
```

Comment:
This is not valid Odoo JS. But why are you adding any logic related to uploading? The ticket specifically said not to & we discussed multiple times in stand-ups...